features/networking/service.feature

```diff
@@ -532,3 +531,60 @@ Feature: Service related networking scenarios
       | type          | merge                                            |
     Then the step should fail
     And the output should contain "does not completely cover the previous range"
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-10216
+  @admin
+  Scenario: The iptables rules for the service should be DNAT or REDIRECT to node after being idled
+    Given I have a project
+    And evaluation of `project.name` is stored in the :proj_name clipboard
+    Given I obtain test data file "networking/list_for_pods.json"
+    When I run oc create over "list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    And I wait until number of replicas match "1" for replicationController "test-rc"
+    Given I use the "test-service" service
+    And evaluation of `service.ip(user: user)` is stored in the :service_ip clipboard
```

Comment:
You can use `service.ip` directly.

The same as below steps.

Reply:
updated. thanks